### PR TITLE
adjust `-j4` for `make`

### PR DESCRIPTION
**Pull request by qin-nz**
Hi,
I am new to P4 and I run the `./install_deps.sh` on my computer.  
The CPU of my computer only have two core.
So `make -j4` will cause an error and without a clarify description.
It cost me many hours to find the problem.

I know remove `-j4` need a longer time to compile.
Is there any good solutions?

### CODE REVIEW

File: install_deps.sh [hunk 1]
```diff
@@ -71,7 +71,7 @@ tar zxvf thrift-0.9.2.tar.gz
 cd thrift-0.9.2
 ./configure
 cd test/cpp ; ln -s . .libs ; cd ../..
-make -j4
+make
```
Comment (cc10512) [inline install_deps.sh]: NPROCS=`grep processor /proc/cpuinfo | wc -l`
make -j${NPROCS}

**Comment by antoninbas** [comment]
Hi. Thanks for your contribution. To accept this pull request, I need to make sure that you have signed the Apache CLA, or that you are part of a p4.org member organization? If you are part of a member organization (university or company), please send me your full name and the organization name by email (antonin@barefootnetworks.com).

**Comment by qin-nz** [comment]
@antoninbas I had already signed membership agreement and sent it to membership@p4.org .

**Comment by antoninbas** [comment]
@qin-nz I have to check with p4.org, would you mind sending me your full name at the above email address? Thanks.

**Comment by qin-nz** [comment]
@antoninbas  Hu Dingyuan

**Pull request merged**